components/director/pkg/graphql/schema.graphql

```diff
@@ -533,6 +620,31 @@ type OneTimeTokenForRuntime implements OneTimeToken {
 	rawEncoded: String
 }
 
+type PackageDefinition {
+	id: ID!
+	name: String!
+	description: String
+	authRequestJSONSchema: JSONSchema
+	auth(id: ID!): APIInstanceAuth
+	auths: [APIInstanceAuth!]!
```

Comment:
`auths` --> maybe `runtimeAuths`?

Reply:
As in the comment above, runtime would be not proper word for that. But instance/usage... or even `apiInstanceAuths` or `apiUsageAuths`... I have mixed feelings for all combinations...